### ISSUE TITLE
[CRASH] Crash on load. Just recently started happening

### ISSUE THREAD

**Issue by rolltidehero** [marticliment/WingetUI#1239]
I don't recall making any changes in the settings. I tried running as admin as well just to see if that helped, but no luck.

Not sure why it says Windows 10.  Here are my Windows specs:

Edition	Windows 11 Pro
Version	22H2
Installed on	‎2/‎10/‎2023
OS build	22621.2070
Experience	Windows Feature Experience Pack 1000.22659.1000.0

I did some minor research on my lunch break and tried to upgrade a couple of python pkgs but still no luck.

BTW — Great friggin' job with this app! It's probably my most important piece of software and that's coming from a software addict! Do you have a Patreon or some type of club membership for something like special feature requests for those who would like to be a part of the team but don't really have the time or even the ability to contribute via pull requests? I know you have the Kofi thing, but I think a Patreon or Locals would be pretty sweet as well. Regardless though, good job dude! Below is the crash report.

Hard crash experienced

Error details : 
```
                        OS: Windows
                   Version: ('10', '10.0.22621', 'SP0', 'Multiprocessor Free')
           OS Architecture: AMD64
          APP Architecture: 64bit
                  Language: Unknown
               APP Version: Unknown
                   Program: WingetUI
           Program section: Main script

-----------------------------------------------------------------------------------------



WingetUI Log:




Traceback (most recent call last):
  File "__init__.py", line 77, in <module>
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File ".\customWidgets.py", line 7, in <module>
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File ".\genericCustomWidgets.py", line 10, in <module>
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File ".\tools.py", line 17, in <module>
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File "clr.py", line 6, in <module>
  File "pythonnet\__init__.py", line 143, in load
  File "clr_loader\types.py", line 64, in __call__

RuntimeError: cannot call null pointer pointer from cdata 'int(*)(void *, int)'<class 'RuntimeError'>: cannot call null pointer pointer from cdata 'int(*)(void *, int)'
```

**Comment by calbrecht** [comment]
Same log here on
```
              OS: Windows
         Version: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
 OS Architecture: AMD64
APP Architecture: 64bit
```

After fresh install of `WingetUI 2.0.1` from Users PowerShell with
```
winget remove wingetui
...
winget install wingetui
```
is starts fine, however after closing the Application Window auto-selfupdates to Version 2.0.2 and crashes on next startup.

**Comment by marticliment** [comment]
Hello,

do you have .NET runtimes installed?

**Comment by marticliment** [comment]
Dear users, 

a pull request with a fix is coming.

Meanwhile, please install WingetUI 2.1.0 beta, that won't get autoupdated to version 2.0.2.
The link to download this release is the following: https://github.com/marticliment/WingetUI/releases/download/2.1.0-beta/WingetUI.Installer.exe

**Comment by rolltidehero** [comment]
> Dear users,
> 
> A fix has been released on beta 2.0.3
> 
> The link to download this release is the following: [`2.0.3-beta` WingetUI.Installer.exe (download)](https://github.com/marticliment/WingetUI/releases/download/2.0.3-beta/WingetUI.Installer.exe)

Thanks! That worked!
